pkg/config/msp_test.go

```diff
@@ -801,7 +774,7 @@ func TestAddRootCAToMSP(t *testing.T) {
 	},
 	"sequence": "0"
 }
-	`
+`, org1CertBase64, org1CertBase64, org1CertBase64, org1CertBase64, org1CertBase64, org1CertBase64, org1CertBase64, org1CRLBase64, org1CertBase64, certBase64, org1PKBase64, org1CertBase64, org1CertBase64, org1CertBase64, org2CertBase64, org2CertBase64, org2CertBase64, org2CertBase64, org2CertBase64, org2CertBase64, org2CertBase64, org2CRLBase64, org2CertBase64, org2PKBase64, org2CertBase64, org2CertBase64, org2CertBase64)
```

Comment:
This works but there’s a lot of repetition. Two alternatives to consider:

- Use indexes in the format string so you can reference the same item. `%[1]s` would be the first argument; you can use that format string multiple times.
- Consider using a template instead of a simple printf. Then you can use a struct with named fields and reference the fields in the template with `{{Org1Cert}}` style names.

Reply:
Ah, I knew there had to be a way built into printf to handle using the same argument multiple times but couldn't figure out the right way to search for that functionality. :) Thanks. Yeah, I didn't like doing it this way but wanted to get something working for now.